src/components/Footer/Footer.tsx

```diff
@@ -7,7 +7,7 @@ import { ReactComponent as Communiti2 } from "../../assets/logos/communiti2.svg"
 
 const Footer = () => {
   return (
-    <div className="footer">
+    <footer className="footer">
```

Comment:
[OPTIONAL] Since you have a ‘footer’ tag already, maybe it’s not necessary to have a ‘footer’ class?

Reply:
Thanks, I'll continue the styling for tailwind